hadoop-gremlin/conf/hadoop-graphson.properties

```diff
@@ -28,14 +28,43 @@ gremlin.hadoop.jarsInDistributedCache=true
 # the vertex program to execute
 gremlin.vertexProgram=org.apache.tinkerpop.gremlin.process.computer.ranking.pagerank.PageRankVertexProgram
 
-####################################
-# SparkGraphComputer Configuration #
-####################################
+############################################
+# SparkGraphComputer Configuration (Local) #
+############################################
 spark.master=local[4]
 
-#####################################
-# GiraphGraphComputer Configuration #
-#####################################
+############################################
+# SparkGraphComputer Configuration (Yarn) #
+############################################
+#spark.master=yarn-client
+# app name in the cluster's history server
+#spark.app.name=TinkerPop
+# faster startup if many cluster users ask for the default 4040 port
+#spark.ui.port=4050
+# needed if the default JAVA_HOME on the cluster is not jdk1.8
+#spark.yarn.appMasterEnv.JAVA_HOME=/usr/lib/jvm/java-1.8.0-oracle.x86_64/jre
+#spark.executorEnv.JAVA_HOME=/usr/lib/jvm/java-1.8.0-oracle.x86_64/jre
+# below point to your cluster's worker node configs
+#spark.executorEnv.SPARK_CONF_DIR=/etc/spark/conf
+#spark.executorEnv.HADOOP_CONF_DIR=/etc/hadoop/conf
+# needed for Hortonworks Data Platform
+#spark.executor.extraJavaOptions=-Dhdp.version=2.3.2.0-2950
+# optional parameters for tuning your TinkerPop application
+#spark.executor.instances=10
+#spark.executor.memory=4g
+#spark.executor.cores=2
+# needed on a secure cluster to allow for multiple computations
+# on a single Kerberos ticket
+#gremlin.spark.persistContext=true
+
+# Below copy all properties from your spark-client's spark-defaults.conf
```

Comment:
Is that necessary? Recommended? The purpose of `*-defaults.conf` files is to set up defaults which you don't want to specify somewhere else, no?